<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_listgcc</class>
 <widget class="QWidget" name="C_listgcc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Элементы комплекса управления</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListWidget" name="listWidget">
        <item>
         <property name="text">
          <string>Средства управления</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Пункты приема информации</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Перечень выбранных объектов</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QListWidget" name="listWidget_2"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="minimumSize">
      <size>
       <width>289</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>289</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Координаты выбранного объекта</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QLabel" name="label_longitude">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Долгота, гр.</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_longitude">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_latitude">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::DefaultContextMenu</enum>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_latitude">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Широта, гр.</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
